production/helm/loki/CHANGELOG.md

```diff
@@ -13,8 +13,17 @@ Entries should include a reference to the pull request that introduced the chang
 
 [//]: # (<AUTOMATED_UPDATES_LOCATOR> : do not remove this line. This locator is used by the CI pipeline to automatically create a changelog entry for each new Loki release. Add other chart versions and respective changelog entries bellow this line.)
 
+## 6.2.0
+
+- [FEATURE] Add a headless service to the bloom gateway component.
+
+## 6.1.0
+
+- [CHANGE] Only default bucket names in helm when using minio.
```

Comment:
Is this correct? Looks like it's from a previous commit

Reply:
it kinda is. the pr that released 6.1.0 didn't add the changelog entry, so I'm backfilling the entry. Not sure we want to keep adding CHANGELOG entries though.